app/src/main/java/com/kickstarter/ui/activities/ProjectActivity.kt

```diff
@@ -623,7 +624,11 @@ class ProjectActivity : BaseActivity<ProjectViewModel.ViewModel>(), CancelPledge
         project_action_button.layoutParams = buttonParams
     }
 
-    private fun toggleManagePledgeVisibility(visible: Boolean) {
-        if (visible) rewards_toolbar.inflateMenu(R.menu.manage_pledge) else rewards_toolbar.menu.clear()
+    private fun updateManagePledgeMenu(@MenuRes menu: Int?) {
```

Comment:
Could we also check the presence of the menu at the callsite of `updateManagePledgeMenu`, and pass down a `Boolean`? 

```
fun updateManagePledgeMenu(menuVisible: Boolean): void

this.updateManagePledgeMenu(menu?)
```

Either way works though!

Reply:
The menu always exists! https://developer.android.com/reference/android/widget/Toolbar.html#getMenu()